flatpak_options.py.patch

```diff
@@ -0,0 +1,23 @@
+--- a/scons/flatpak_options.py	2024-09-06 15:44:34.326102896 -0500
++++ n/scons/flatpak_options.py	2024-09-06 15:50:41.539115973 -0500
+@@ -32,13 +32,17 @@
+ 
+ cxx_extra='-w -O3'
+ 
+-pythoncmd='/app/bin/python3'
+-pythonlibpath = '/app/lib'
+-pythonincpath = '/app/include/python3.7m'
++pythoncmd='python3'
++pythonlibpath = '/app/lib/python3.10'
++pythonincpath = '/app/include/python3.10'
++pythonlibname='python3.10'
```

Comment:
23.08 has python 3.11

Reply:
Using this version breaks some dependencies. I can try to compile version 6.0 with python 3.11, but it's hard to know if I'll be successful. I think this version should be merged anyway.